app/views/layouts/application.slim

```diff
@@ -58,19 +58,20 @@ html
                   = button_tag(type: 'submit', class: 'btn btn-primary', name: nil) do
                     = icon('fas', 'search')
           ul.nav.navbar-nav
+            li.nav-item
```

Comment:
Previously the patreon link vanishes if you are a patreon, and this will change it to always visible.  Is this correct behavior?

Reply:
Yeah this is intentional, but I could be convinced either way. On one hand it's a simpler user experience to have the same buttons always there and not vanish for an apparently unknown reason. On the other hand you could see the button as promoting Patreon and if you're a patron it's a useless distraction. It's a toss-up.

Reply:
I think with the button being extracted to the main header from the dropdown, having the button always there is better.  That way the header always has a consistent look.